src/test/groovy/com/ofg/uptodate/reporting/NewVersionLoggerSpec.groovy

```diff
@@ -0,0 +1,55 @@
+package com.ofg.uptodate.reporting
+
+import com.ofg.uptodate.LoggerProxy
+import com.ofg.uptodate.dependency.Dependency
+import spock.lang.Specification
+
+import static com.ofg.uptodate.reporting.NewVersionLogger.NEW_VERSIONS_AVAILABLE
+import static com.ofg.uptodate.reporting.NewVersionLogger.NEW_VERSIONS_MESSAGE_HEADER
+import static com.ofg.uptodate.reporting.NewVersionLogger.NO_NEW_VERSIONS_AVAILABLE
+import static com.ofg.uptodate.reporting.NewVersionLogger.NO_NEW_VERSIONS_MESSAGE
+
+class NewVersionLoggerSpec extends Specification {
+
+    static final Dependency HIBERNATE = new Dependency('org.hibernate', 'hibernate-core', '4.3.6.Final')
```

Comment:
I doesn't change much, but maybe old those constants and fields could be `private`?